src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +51,10 @@ public static RobotContainer getInstance() {
         return INSTANCE;
     }
 
-    private void configureDefaultCommands() {}
+    private void configureDefaultCommands() {
+        intake.setAngle(Units.Degrees.of(0).mutableCopy());
```

Comment:
Remember, `setAngle` returns a command, but what do you do with it?

Reply:
right......

Reply:
423a178e32ada435b677b785a6c731fd229beeee